### PR TITLE
Typos and Playground fixes

### PR DESCRIPTION
**Pull request by kirillyakimovich**

### CODE REVIEW

File: Playgrounds/SimpleStorage.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+let storage = try! Storage(diskConfig: diskConfig, memoryConfig: memoryConfig)
+
+let user = User(id: 1, firstName: "John", lastName: "Snow")
+let key = "\(user.id)"
+
+// Add objects to the cache
+try storage.setObject(user, forKey: key)
+
+// Fetch object from the cache
+storage.async.object(ofType:User.self, forKey: key) { result in
```
Comment (onmyway133) [inline Playgrounds/SimpleStorage.playground/Contents.swift]: Would you mind putting a space after `:` to match our convention ?

**Comment by onmyway133** [comment]
@kirillyakimovich Wow! This is cool. Thanks for updating all the playgrounds. I've added a comment asking if you mind putting a space after `:` in your PR, to match our convention. Besides that, it looks great to me 👍

File: Playgrounds/SimpleStorage.playground/Contents.swift (hunk 1 above)
Comment (kirillyakimovich) [reply Playgrounds/SimpleStorage.playground/Contents.swift]: Sure, sorry for that :)

**Comment by kirillyakimovich** [comment]
@onmyway133 Thank you! Done!

**Comment by onmyway133** [comment]
@kirillyakimovich You're awesome 👍

**Pull request merged**